src/pybind/mgr/dashboard/frontend/src/app/core/navigation/dashboard-help/dashboard-help.component.html

```diff
@@ -0,0 +1,26 @@
+<div dropdown>
+  <a dropdownToggle
+     class="dropdown-toggle"
+     data-toggle="dropdown">
+    <i class="fa fa-question-circle-o"></i>
+    <span i18n>Help</span>
+    <span class="caret"></span>
+  </a>
+  <ul *dropdownMenu
+      class="dropdown-menu">
+    <li>
+      <a i18n
+         class="dropdown-item"
+         href="http://docs.ceph.com/docs/mimic/mgr/dashboard/"
```

Comment:
This URL is pointing to the mimic branch, but actually the code in master should already point to nautilus. This will create a situation hard to maintain, because every backport needs to change this.

Is it possible to get the current cluster version from some endpoint and generate this URL dynamically? that way the code is the same independently of the version.

Reply:
@rjfd Summary endpoint will now return the ceph version, and the documentation URL will be created based on the release name.